python/spear/engine_service.py

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import spear
+import sys
+
+# TODO (RP): Separate EngineService and GameWorldService function calls from this class.
+class EngineService():
+    def __init__(self, rpc_client):
+        self._rpc_client = rpc_client
+
+    def call(self, service_name, func_name, *args):
```

Comment:
Remove this `call` function. If someone really wants to do this, they can do it by directly accessing `_rpc_client`.